subtests/docker_cli/run/run.py

```diff
@@ -43,6 +43,7 @@ def init_subargs(self):
             self.sub_stuff['containers'].append(name)
             subargs.append('--name')
             subargs.append(name)
+            self.sub_stuff['expected_name'] = name
```

Comment:
Probably don't need this as long as you're only working with a single container, you can reference it by ``self.sub_stuff['containers'][-1]``